### PR TITLE
Remove todo comments, or implement changes as suggested in comments

### LINKED ISSUE #393: Remove todo-comments in code
**Todays situation:**
In Argus-frontend code base, there are code bits/functions that are marked with comments suggesting that some code is not finished, or need polish/refactoring. 

**Suggested solution:**
Comments pointing out not-finished code are not many. Additionally, upon further inspection, it is apparent that some of those comments are not relevant anymore.
Therefore, the suggested solution is to:

- **either** finish/polish the code bit as suggested in the comments
- **or** remove the comment if changes are not relevant anymore

**Credit:**
This issue was recommended by GÉANT in _Argus code audit_ [July 2022]. 

**Additional info:**
All files (including line numbers) where this issue is present are listed in _Argus code audit_.

### PR DESCRIPTION
**Pull request by podliashanyk**
Go through unfinished code as suggested in _Argus code audit_ [July 2022] by GÉANT.

Closes #393

### CODE REVIEW

File: src/services/RealtimeService.ts [hunk 1]
```diff
@@ -212,7 +212,7 @@ export class RealtimeService {
     this.ws.onerror = (e: any) => {
       console.error(`[RealtimeService ${this.id}] got error on websocket client, closing: ${e.message}`);
       if (this.ws) this.ws.close();
-      // this.setState("closed"); // TODO: should this be set?
+      this.setState("closed"); 
     };
```
Comment (stveit) [inline src/services/RealtimeService.ts:216]: Is this the same change as in PR #402 ?

File: src/views/incident/IncidentDetailView.tsx [hunk 2]
```diff
@@ -29,9 +29,7 @@ export const IncidentDetailsView: React.FC<IncidentDetailsViewPropsType> = () =>
         displayAlert(`Failed to get incident with pk=${pk}`, "error");
         setIncident(null);
       });
-    // FIXME:
-    // eslint-disable-next-line
-  }, [pk]);
+  }, [pk, displayAlert]);
```
Comment (stveit) [inline src/views/incident/IncidentDetailView.tsx:32]: Where does this `displayAlert` come from? does not seem to be relevant to any comments?

File: src/api/client.ts [hunk 3]
```diff
@@ -438,8 +424,6 @@ class ApiClient {
     );
   }
 
-  // NOTE: This won't actually get all incidents anymore because of the pagination
```
Comment (stveit) [inline src/api/client.ts:441]: seems like a reasonable comment tbh, name is misleading and the comment corrects it (ASSUMING that this comment is even true, I have no idea. if its not true, then ofc remove it). Best would be to make the name less misleading, but since the next comment is "// TODO: Remove all use of this method.", its probably not worth putting energy into renaming if its gonna be removed soon.

**Review by stveit — COMMENTED** [review]
comments based on ignorance of the codebase

File: src/services/RealtimeService.ts (hunk 1 above)
Comment (podliashanyk) [reply src/services/RealtimeService.ts:216]: Yes

File: src/views/incident/IncidentDetailView.tsx (hunk 2 above)
Comment (podliashanyk) [reply src/views/incident/IncidentDetailView.tsx:32]: That's what `FIXME` refers to. Without _displayAlert_ in dependencies, the program will not compile when `// eslint-disable-next-line` is removed

File: src/api/client.ts (hunk 3 above)
Comment (podliashanyk) [reply src/api/client.ts:441]: This comment is outdated

File: src/views/incident/IncidentDetailView.tsx (hunk 2 above)
Comment (stveit) [reply src/views/incident/IncidentDetailView.tsx:32]: aaah okay, its some react shenanigans with useEffect

**Review by stveit — APPROVED** [review]
guess i dont have anything more to say other than 👍
